processor/transformprocessor/internal/metrics/func_convert_gauge_to_sum_test.go

```diff
@@ -0,0 +1,160 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+)
+
+func Test_convert_gauge_to_sum(t *testing.T) {
```

Comment:
I'm not sure there's really a benefit to matching the test name style (snake case). It should be easy enough to correlate using normal syntax.

Reply:
@djaglowski updated test function names